### PR TITLE
Import CI build drivers

### PR DESCRIPTION
**Pull request by zeha**
These are our new build scripts for driving grml-live from CI. Not the nicest Python I've ever written, but it works.

This PR also contains #172 and #173 for now, as I needed them for testing in the same branch.

### CODE REVIEW

File: grml-live [hunk 1]
```diff
@@ -637,32 +637,29 @@ extract_iso() {
 if [ -n "$EXTRACT_ISO_NAME" ]; then
   log "Unpacking ISO from ${EXTRACT_ISO_NAME}"
   einfo "Unpacking ISO from ${EXTRACT_ISO_NAME}"
-  local mountpoint=$(mktemp -d)
+  local tempdir=$(mktemp -d)
   local rc=0
-  mount -o loop "${EXTRACT_ISO_NAME}" "$mountpoint" ; rc=$?
+  mkdir -p "${tempdir}/live/"
+  osirrox -indev "${EXTRACT_ISO_NAME}" -extract live "${tempdir}/live/"
```
Comment (mika) [inline grml-live:643]: missing the "; rc=$?" at the end to propagate the exit code for lines below?

Comment (mika) [inline grml-live:643]: TIL about osirrox, nice! :+1:

File: grml-live [hunk 2]
```diff
@@ -637,32 +637,29 @@ extract_iso() {
...
   if [ "$rc" != 0 ]; then
-    rmdir "$mountpoint"
-    log "mount failed"
-    eerror "mount failed"
+    rm -r "$tempdir"
```
Comment (mika) [inline grml-live:645]: are we good/fine with `rm -r` (as in: no need for `rm -rf`)?

**Review by mika — COMMENTED** [review]
Wow! :bow:

File: grml-live (hunk 1 above)
Comment (zeha) [reply grml-live]: indeed, fixed now!

File: grml-live (hunk 2 above)
Comment (zeha) [reply grml-live]: Was pondering that; have now added -f.

**Review by mika — APPROVED** [review]
Can't really comment on the python thingy, but everything LGTM!
:hugs:

**Pull request merged**